hash/ibm5150.xml

```diff
@@ -13168,6 +13338,65 @@ has been replaced with an all-zero block. -->
 		</part>
 	</software>
 
+	<software name="nshift">
+		<description>Night Shift (5.25")</description>
+		<year>1990</year>
+		<publisher>Lucasfilm</publisher>
+		<info name="version" value="V1.1" />
+		<info name="developer" value="Attention to Detail" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="368640">
+				<rom name="Night Shift [Lucasfilm] [1990] [5.25DD] [Disk 1 of 3].img" size="368640" crc="e4f1af89" sha1="93013386d3ec6ed9479077b84366b90de818fecf"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_5_25">
+			<dataarea name="flop" size="368640">
+				<rom name="Night Shift [Lucasfilm] [1990] [5.25DD] [Disk 2 of 3].img" size="368640" crc="eeb79a13" sha1="02adbf9824b520d2f6b65e4188a631003531cfb8"/>
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_5_25">
+			<dataarea name="flop" size="368640">
+				<rom name="Night Shift [Lucasfilm] [1990] [5.25DD] [Disk 3 of 3].img" size="368640" crc="ba0a19c2" sha1="1f66ed76630a61181db05df5d74935fb597daf3c"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nshift_35" cloneof="nshift">
+		<description>Night Shift (3.5")</description>
+		<year>1990</year>
+		<publisher>Lucasfilm</publisher>
+		<info name="developer" value="Attention to Detail" />
+		<info name="version" value="V1.1" />
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size = "737280">
+				<rom name="Night Shift [Lucasfilm] [1990] [3.5DD] [Disk 1 of 2].img" size="737280" crc="291faf52" sha1="32b2162aa9d0da03b20d852e6d81d7dffc89c30f"/>
+			</dataarea>
+		</part>
+		<part name="flop1" interface="floppy_3_5">
```

Comment:
"flop1" should be "flop2" to avoid duplication error

Reply:
Thanks for the catch! Correction done.